openshift/values-patch.yaml

```diff
@@ -47,6 +47,8 @@ identity:
 
 # omit the values below if elasticsearch.enabled is false
 elasticsearch:
+  tests:
```

Comment:
If we just disable it because of the bug in the post-render and webhook ([#7891](https://github.com/helm/helm/issues/7891)), then I'd suggest adding a comment here and link for the bug.

If we want to disable all test manifests in general because they are not needed in prod, then we also need to disable the other tests like for Zeebe.

Reply:
I thought the same whether we want to remove that from our production values file as well. But not sure whether people run helm test from time to time :shrug: There must be a purpose why elastic has this per default on :D

Reply:
yeah, for that reason, I'm in favor of just documenting it by adding the comment there with the link because it will always fail when it's applied (because of the bug).